hail/python/test/hail/backend/test_service_backend.py

```diff
@@ -1,8 +1,13 @@
+import os
+
 import hail as hl
 
 from ..helpers import skip_unless_service_backend
 from hail.backend.service_backend import ServiceBackend
 
+CLOUD = os.environ['HAIL_CLOUD']
```

Comment:
Does this impede running other query tests locally? Not sure if it does but it would be annoying to have to specify `HAIL_CLOUD` to run tests that have nothing to do with it. I'd prefer in that case to put the `os.environ` call in the method that actually needs it and remove the env variable from test steps that aren't testing the service backend.